src/common/replayBrowser/ReplayClient.ts

```diff
@@ -0,0 +1,99 @@
+import { ipcRenderer } from "electron";
+
+import { FileLoadResult, FileResult, FolderResult } from "./types";
+
+export async function loadReplays(folder: string, progressCallback: (count: number, total: number) => void) {
```

Comment:
These event handlers should be refactored to use `electron-better-ipc`.

Reply:
I checked out the library, it looks nice, but it seems oriented to request/response, not sure how to handle the callback messages to update the progress. Did you have something in mind?

Reply:
nvm I see you already have it on your branch